crates/bevy_ui/src/ui_node.rs

```diff
@@ -347,6 +347,20 @@ impl From<Vec2> for ScrollPosition {
     }
 }
 
+/// Determines which axes of a node are *sticky* during scrolling.
+///
+/// A **sticky** node maintains its position along the specified axes
+/// instead of moving with its scrolled parent content when [`ScrollPosition`] is applied.
+#[derive(Component, Debug, Clone, Default, Deref, DerefMut, Reflect)]
+#[reflect(Component, Default, Clone)]
+pub struct ScrollSticky(pub BVec2);
+
+impl From<BVec2> for ScrollSticky {
+    fn from(value: BVec2) -> Self {
+        Self(value)
+    }
+}
```

Comment:
I don't like the plain `BVec2` here, I think it should have named fields or maybe some helpers like:
```rust
impl ScrollSticky {
    fn left() -> Self { 
        Self(BVec2::new(true, false))
    }
    
    // etc..
}
```

Reply:
I would like to avoid implying in API that ScrollSticky has method `left`. What does it mean?

Sticky headers can be achieved with this feature only by combining multiple features:
ZIndex, BackgroundColor, ScrollSticky, parent with display Grid.

Users need to carefully mix multiple features to get the final behavior working. So I would like to avoid implying any specific use case in this level of API.

See example: https://github.com/bevyengine/bevy/pull/21648/commits/5ab4c9a070a381da189344d77a708385084c1c02